Makefile

```diff
@@ -41,6 +41,11 @@ all: jlm-release check
 .PHONY: release
 release: jlm-release
 
+.PHONY: release-gcc
+release-gcc: CXX=g++ 
+release-gcc: jlm-release
+release-gcc: check
+
```

Comment:
There is no need to introduce a new make target. Just use make CXX=g++ release -j2

Reply:
This commit has been removed.